src/thunderbots/software/geom/util.cpp

```diff
@@ -70,6 +70,27 @@ double dist(const Segment &first, const Vector &second)
     return dist(second, first);
 }
 
+double dist(const Point &first, const Polygon &second)
```

Comment:
I won't make this blocking, as we do it elsewhere, but could we perhaps use names like `point` and `polygon` instead of `first` and `second`?

Reply:
I'll address this in some later geom cleanup. It's a problem nearly everywhere in this file.